test/unit/test.rdf.js

```diff
@@ -365,13 +366,17 @@ describe("lib/rdf", function() {
       expect(locale.childNodes[3].childNodes[0].data).to.be.equal("名前");
       expect(locale.childNodes[5].tagName).to.be.equal("em:description");
       expect(locale.childNodes[5].childNodes[0].data).to.be.equal("紹介");
+      expect(locale.childNodes[7].tagName).to.be.equal("em:creator");
```

Comment:
I don't see `em:creator` or `author` used in this test. Is it necessary to make an assertion about it?